grails-app/assets/javascripts/reporting.js

```diff
@@ -580,32 +590,19 @@ var GrantManagerReportsViewModel = function(config) {
     });
 
     self.generateProjectReports = function () {
-        var jsData = {
-            plannedStartDate: self.plannedStartDate(),
-        };
-
-        var startDateSelector = "#generate-report input[data-bind*=plannedStartDate]";
-
-        var message;
-        if (!self.plannedStartDate()) {
-            message =  "The planned start date is a required field";
-        }
-        if (self.plannedStartDate() >= config.project.plannedEndDate) {
-            message =  "The project start date must be before the end date";
-        }
-
-        if (message) {
-            setTimeout(function() {
-                $(startDateSelector).validationEngine("showPrompt", message, "topRight", true);
-            }, 100);
-
-        } else {
+        $('#reportingTabDatesForm').validationEngine();
```

Comment:
Please refactor the hardcoded selector here into a config item (e.g. config.datesFormSelector).